stablehlo/reference/Ops.cpp

```diff
@@ -402,6 +406,52 @@ Tensor evalAtan2Op(const Tensor &lhs, const Tensor &rhs,
   return result;
 }
 
+Tensor evalBitcastConvertOp(const Tensor &operand, ShapedType resultType) {
+  Tensor result(resultType);
+  auto operandElementType = operand.getElementType();
+  auto resultElementType = result.getElementType();
+  int64_t operandBitWidth = isSupportedComplexType(operandElementType)
```

Comment:
complex type are not supported right?

Reply:
Conversions between complex and non-complex types aren't supported but complex to complex is supported.